src/main/resources/static/styles/style.css

```diff
@@ -570,10 +570,6 @@ table.striped>tbody>tr:nth-child(odd) {
 	margin-top: 10px;
 }
 
-.emoji-picker-container * {
-	z-index: 10;

```

Comment:
This is kind of important because in some places there is conflict between the emoji picker and other UI components. For example, when editing question in full screen mode.

Reply:
I corrected by reducing the selector in new commit